Gemfile

```diff
@@ -16,19 +16,17 @@ gem 'rails', '~> 6.0.2', '>= 6.0.2.2'
 gem 'sass-rails', '>= 6'
 gem 'turbolinks', '~> 5'
 gem 'webpacker', '~> 4.0'
-gem 'sendgrid_actionmailer_adapter'
+# gem 'sendgrid_actionmailer_adapter'
 gem 'strong_migrations'
 gem 'csv'
-
-group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```

Comment:
I use `byebug` in development too, it'd be great if it could stay there

Reply:
me too! it's still in there, i just moved it into the other development/test group further down the file